tests/test_array.py

```diff
@@ -1382,3 +1384,39 @@ def test_roundtrip_numcodecs() -> None:
     metadata = root["test"].metadata.to_dict()
     expected = (*filters, BYTES_CODEC, *compressors)
     assert metadata["codecs"] == expected
+
+
+def _index_array(arr: Array, index: Any) -> Any:
+    return arr[index]
+
+
+@pytest.mark.parametrize(
+    "method",
+    [
+        pytest.param(
+            "fork",
+            marks=pytest.mark.skipif(
+                sys.platform in ("win32", "darwin"), reason="fork not supported on Windows or OSX"
```

Comment:
Because windows should only perform spawn? This decorator is a bit verbose, it would be OK to put `if ... : pytest.skip()` in the body of the function.

Reply:
I think I prefer (verbose) parametrization over checking the OS in the test itself. The former gives better feedback in the test summary.